### PR TITLE
Beta feedback 2022-06-03

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_5C.txt [hunk 1]
```diff
@@ -265,7 +265,7 @@ Arcueid doesn't seem to feel like sitting down for whatever reason,
 -- Page 521, Offset 25613. Mods: @n.
 -- 時間だけがすぎていく
 // 'Only time passes' as in nothing else happens, I assume
-Everything else is still save for the hands of the clock
+Everything else is still, save for the hands of the clock
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_5C.txt:268]: Don't think I like this comma but I would drop the `else`

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 2]
```diff
@@ -134,7 +134,7 @@ I don't know how many years it'll be before I wake up again,
 // Leading space for glue
 // 損をして is like 'doing so at a loss' (e.g. selling),
 // interpreted as a loss of the potential afforded by acclimatizing to the period
- But it feels like by doing so I'm losing out.
+ But it feels like by doing so, I'm losing out.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:137]: i think this one is unnecessary

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 3]
```diff
@@ -467,7 +467,7 @@ Then... does that mean she derives sustenance from the same sorts of food that p
 [sha:4ecbcb904506102ea5903304be773082a19e137a]{
 -- Page 329, Offset 25085.
 -- 「なによ、いじわる」
-"What do you want, you bully."
+"What do you want, you bully?"
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:470]: i dont think this comma is really necessary

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 4]
```diff
@@ -921,7 +921,7 @@ I don't like what-ifs.
 -- Page 357, Offset 25169.
 -- どんな結果になるか分からないけど、とりあえずその時は、どこかに救いがあるような気がするでしょう？」
 You might not know where you're gonna end up,
- but in the moment you always feel like there's some kind of salvation out there, right?"
+ but in the moment, you always feel like there's some kind of salvation out there, right?"
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:924]: hmm, i think this one reads more naturally as a single, uninterrupted thought after the first clause. i say revert

File: script/Arcueid/Day 10/10_00_ARC10_5C.txt (hunk 1 above)
Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_5C.txt:268]: personally I'd keep the comma and drop the "else"
> Everything is still, save for the hands of the clock

is how i'd naturally pause it if reading for dramatic effect

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 5]
```diff
@@ -370,7 +370,7 @@ At no point did I feel like Arcueid really was a vampire.
 [sha:e5ce3e599a0ad6f3b66afea81fce974d98f8cf7f]{
 -- Page 582, Offset 25781.
 -- ――――どんな答えを期待してるんだ、おまえは。
-――――What kind of answer are you expecting, you moron.
+――――What kind of answer are you expecting, you moron?
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:373]: dunno the va here but i think this one might wanna stay a period

**Review by Avuxo — APPROVED** [review]

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt (hunk 5 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_5D.txt:373]: Not voiced but I'm fine with periods

**Pull request merged**